src/fidesops/api/v1/endpoints/connection_type_endpoints.py

```diff
@@ -0,0 +1,48 @@
+import logging
+from typing import List, Optional
+
+from fastapi import APIRouter, Depends
+from fastapi.params import Security
+from fastapi_pagination import Page, Params, paginate
+from fastapi_pagination.bases import AbstractPage
+
+from fidesops.api.v1.scope_registry import CONNECTION_TYPE_READ
+from fidesops.api.v1.urn_registry import CONNECTION_TYPES, V1_URL_PREFIX
+from fidesops.models.connectionconfig import ConnectionType
+from fidesops.schemas.saas.saas_config import SaaSType
+from fidesops.util.oauth_util import verify_oauth_client
+
+router = APIRouter(tags=["Connection Types"], prefix=V1_URL_PREFIX)
+
+logger = logging.getLogger(__name__)
+
+
+@router.get(
+    CONNECTION_TYPES,
+    dependencies=[Security(verify_oauth_client, scopes=[CONNECTION_TYPE_READ])],
+    response_model=Page[str],
+)
+def get_all_connection_types(
+    *, params: Params = Depends(), search: Optional[str] = None
+) -> AbstractPage[str]:
+    """Returns a list of connection options in Fidesops - includes only database and saas options here."""
+
+    def is_match(elem: str) -> bool:
+        """If a search query param was included, is it a substring of an available connector type?"""
+        return search in elem if search else True
+
+    database_types: List[str] = [
+        conn_type.value
+        for conn_type in ConnectionType
+        if conn_type
+        not in [ConnectionType.saas, ConnectionType.https, ConnectionType.manual]
```

Comment:
my assumption is that these types and the "custom" type below shouldn't be included in this list, that these are connectors that are available and ready to go